api.md

```diff
@@ -840,7 +842,127 @@ SELECT reorder_chunk('_timescaledb_internal._hyper_1_10_chunk', 'conditions_devi
 runs a reorder on the `_timescaledb_internal._hyper_1_10_chunk` chunk using the `conditions_device_id_time_idx` index.
 
 ---
+## Continuous Aggregates :community_function: [](continuous-aggregates)
+TimescaleDB allows users the ability to automatically recompute aggregates 
+at predefined intervals and materialize the results. This is suitable for 
+frequently used queries.
+
+## CREATE VIEW 
+`CREATE VIEW` statement is used to create continuous aggregates.
+
+The syntax is:
+``` sql
+    CREATE VIEW <view_name> [ (column_name [, ...] ) ] 
+    WITH ( timescaledb.<option> = <value> )
+    AS
+    <select-query>
+    
+    <select_query> is of the form :
+    SELECT <grouping-cols>, <aggregate-functions>
+    FROM <hypertable>
+    [WHERE ... ]
+    GROUP BY <time_bucket( <const-value>, partition-col-of-hypertable>),
+             [ optional grouping exprs>
+    [HAVING ...]
+```
+
+#### Parameters
+|Name|Description|
+|---|---|
+| `<view_name>` | Name (optionally schema-qualified) of continuous aggregate view to be created.|
+| `<column_name>`| Optional list of names to be used for columns of the view. If not given, the column names are deduced from the query.|
+ | `WITH` clause | This clause specifies [options](#create-view-with) for the continuous aggregate view.|
+ | `<select_query>`| A `SELECT` query that uses the specified syntax. |
+
+#### `WITH` clause options [](create-view-with)
+|Name|Description|
+|---|---|
+|**Required**||
+ `timescaledb.continuous`|If timescaledb.continuous is not specified, then this is a regular postgres view. |
```

Comment:
Needs a `|` at the start of the line for correct table format


<img width="871" alt="Screen Shot 2019-05-02 at 12 20 24 PM" src="https://user-images.githubusercontent.com/12288273/57090526-fde4af00-6cd4-11e9-9560-c02223aa80cd.png">

Reply:
Also, we try to always refer to postgres as "PostgreSQL" in the docs

Reply:
fixed.